recipes/pytalises/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "pytalises" %}
+{% set version = "0.1.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytalises-{{ version }}.tar.gz
+  sha256: 880432c41ef1b7bff66b48a370cd596f6fdf8b4290cbb18c1e2df873e5f84576
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
```

Comment:
Under `host`, I believe you only need `python >=3.6` and `pip` since the others aren't needed to build the package and aren't imported directly or indirectly in the `setup.py`.

Reply:
yes. changed in b60dd132f2f5dcbc4abcff5320d0c12e03d1035c